### PR TITLE
initialized react project

### PR DESCRIPTION
**Pull request by jeinygroove**
This is react project, please try to run in on you laptop with `mvn package appengine:run` and check if it's working (you should go to `http://localhost:8080` and see `Hi!`, also in inspector\network you shouldn't see any errors and `hello` request should be successful and return the response `Alpollo says Hi! to the world`).

### CODE REVIEW

**Comment by jeinygroove** [comment]
@googlebot I signed it!

**Comment by jeinygroove** [comment]
@googlebot I signed it!

File: src/main/webapp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (sashapff) [inline src/main/webapp/.gitignore]: You can remove this comment

File: src/main/webapp/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
```
Comment (sashapff) [inline src/main/webapp/public/index.html]: nit: unnecessary comments

File: src/main/webapp/src/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import * as serviceWorker from './serviceWorker';
+
+fetch("/hello").then(() => {ReactDOM.render(
+  <React.StrictMode>
+    <p>Hi!</p>
+  </React.StrictMode>,
+  document.getElementById('root'))});
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
```
Comment (sashapff) [inline src/main/webapp/src/index.js]: nit: unnecessary comments

File: src/main/webapp/src/serviceWorker.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
+// This optional code is used to register a service worker.
+// register() is not called by default.
+
+// This lets the app load faster on subsequent visits in production, and gives
```
Comment (sashapff) [inline src/main/webapp/src/serviceWorker.js]: nit: unnecessary comments

File: src/main/webapp/src/setupTests.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
```
Comment (sashapff) [inline src/main/webapp/src/setupTests.js]: nit: unnecessary comments

File: src/main/webapp/src/index.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import * as serviceWorker from './serviceWorker';
+
+fetch("/hello").then(() => {ReactDOM.render(
```
Comment (sashapff) [inline src/main/webapp/src/index.js]: Add space between '{' and body or delete brackets.

**Comment by jeinygroove** [comment]
Tried to add lighthouse ci (it helps to check the performance of the web app), it works locally, but fails here (maybe it wants some docker things). I will go through this later.

Update: it's in `lighthouse-ci` branch and it works extremely slow and unstable, better to use chrome extension manually instead, ahah

**Review by sashapff — APPROVED** [review]

**Review by RazvanBudacaTechquarter — APPROVED** [review]
LGTM

File: src/main/webapp/public/manifest.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    {
+      "src": "logo192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "logo512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
```
Comment (amaurymedeiros) [inline src/main/webapp/public/manifest.json:19]: Make sure you are using icons/logos with a valid license (better yet if you created your own)

File: src/main/webapp/index.html [hunk 8]
```diff
@@ -1,15 +0,0 @@
-<!DOCTYPE html>
```
Comment (amaurymedeiros) [inline src/main/webapp/index.html]: This file is deleted now, but in https://github.com/googleinterns/step217-2020/pull/1/files we modified it to include the login logic. Make sure to rebase this PR to include the login functionality

File: src/main/webapp/public/manifest.json (hunk 7 above)
Comment (jeinygroove) [reply src/main/webapp/public/manifest.json:19]: This was added by create-react-app (free for commercial use, https://github.com/facebook/react/), it's the part of the default project, so I think the icons are also free, but of course I'm going to do new ones

Comment (amaurymedeiros) [reply src/main/webapp/public/manifest.json:19]: Awesome. They seem to be free for commercial use as well (https://github.com/facebook/react/blob/master/LICENSE), so feel free to use them int this pull request and add the new icons only later

Comment (jeinygroove) [reply src/main/webapp/public/manifest.json:19]: Done in https://github.com/googleinterns/step217-2020/pull/5

File: src/main/webapp/index.html (hunk 8 above)
Comment (jeinygroove) [reply src/main/webapp/index.html]: Done in https://github.com/googleinterns/step217-2020/pull/5

**Review by amaurymedeiros — APPROVED** [review]

**Pull request merged**